server/mongo/src/query.ts

```diff
@@ -44,7 +50,19 @@ export function toMongoQuery<T extends Doc> (
 
   // Find by all classes.
   if (classes.length > 1) {
-    mongoQuery._class = { $in: classes.map((cl) => cl as Ref<Class<Doc>>) }
+    mongoQuery._class = { $in: classes.map((cl) => cl) }
   }
+
+  console.log(mongoQuery)
   return mongoQuery
 }
+
+function translateQuery<P extends keyof T, T extends Doc> (value: QuerySelector<P>): any {
+  const result: any = {
+  }
+  if (value.$in !== undefined) result.$in = value.$in
```

Comment:
I think you just need to copy value selector and update $like if present. In this case in future it potentially will be not required to update this piece of code.